content/post/2020/02/art-design-guidelines/index.md

```diff
@@ -0,0 +1,98 @@
+# GDQuest Art Guideline
+
+In this guideline we aim to establish a visual language that communicates meaningful aspects for our game development teaching content. The art direction is mainly oriented towards flat design principles and focus on high distinguishability of the elements.
+
+Below you'll be presented with graphically rich examples and a brief of the core design behind the use of the colors, shapes and types.  
+
+# Colors
```

Comment:
Don't use level 1 headings for subheadings. This is the title of the page: `# ... #`. Also in hugo, it's the website engine that inserts the page title.